internal/compiler-interface/src/main/java/xsbti/compile/ExternalHooks.java

```diff
@@ -1,12 +1,31 @@
 package xsbti.compile;
 
+import java.io.File;
+
 /**
  * Created by krzysztr on 02/08/2016.
  */
 public interface ExternalHooks {
     public static interface Lookup {}
 
-    public static interface ClassFileManager{}
+    /**
+     * During an incremental compilation run, a ClassfileManager deletes class files and is notified of generated class files.
+     * A ClassfileManager can be used only once.
+     */
+    public static interface ClassFileManager {
```

Comment:
Let's move this class out of `ExternalHooks` if it's being used outside of ExternalHooks.